ble_controller/CHANGELOG.rst

```diff
@@ -5,6 +5,29 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+Added
```

Comment:
Going forward, a new header will only be added for a new NCS tagged release. S.t. the list will contain the changes since the previous tagged version of NCS.